File: 1. Two Sum/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    int i, j;
    int *result = malloc(sizeof(int)*2);
    *returnSize = 0;
    
    for(i=0; i<numsSize; i++) {
        for(j=i+1; j<numsSize; j++) {
            if(nums[i]+nums[j] == target) {
                result[0] = i;
                result[1] = j;
                *returnSize  = 2;
                
                return result;
            }
        }
    }
    
    return result;
    
}

File: 1. Two Sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, _ in enumerate(nums):
            for j, _ in enumerate(nums[i+1:]):            
                if nums[i]+nums[j+i+1] == target:
                    return [i, j+i+1]

        return []

    def twoSumHash(nums, target):
        comps = {}
        for i, val in enumerate(nums):
            comp = target-val
            if comp not in comps:
                comps[val] = i
            else:
                return [comps[comp], i]
